src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/utils/getFeedbackTabs.tsx

```diff
@@ -0,0 +1,31 @@
+import { DatavizActivity } from 'modules/Dashboard/api';
+
+import { AssessmentActivityItem } from '../../RespondentDataReview.types';
+import { FeedbackNotes } from '../FeedbackNotes';
+import { FeedbackReviews } from '../FeedbackReviews';
+
+export const getFeedbackTabs = (
```

Comment:
Could you please re-write this code via react component to use reconciliation mechanism due to performance improvements?